oauth2server/proj/settings/local.py

```diff
@@ -20,7 +20,7 @@
     }
 }
 
-DEBUG = True                                                # Set to false on Production
+DEBUG = os.environ.get('DEBUG','False')
```

Comment:
This one can probably be True by default

Reply:
yeah fair point, will fix

Reply:
fixed